pootle/core/views.py

```diff
@@ -778,7 +778,7 @@ def get_context_data(self, *args, **kwargs):
             raise Http404(
                 ValidationError(search_form.errors).messages)
 
-        total, start, end, units_qs = search_backend.get(Unit)(
+        total, __, __, units_qs = search_backend.get(Unit)(
```

Comment:
My rationale was simple.  `__` makes it much more readable then `something_` so I'm aiming for readability.  The only reason to keep vars is if the function creating them is not that well known and you want to keep that in the code.